crates/net/network/src/transactions.rs

```diff
@@ -1204,6 +1222,11 @@ enum TransactionsCommand {
         peers: Vec<PeerId>,
         tx: oneshot::Sender<HashMap<PeerId, HashSet<TxHash>>>,
     },
+    /// Request specific transactions from a peer.
+    GetPeerSender {
+        peer_id: PeerId,
```

Comment:
that makes me think maybe we should remove `peer_id` 'cause it already exists on `PeerRequestSender`